### PR TITLE
Ma/#856 beamville csv setup 4ci

### PR DESCRIPTION
**Pull request by mr-asif-nor**
Urbansim related csv based run.

---
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1128)

### CODE REVIEW

**Comment by mr-asif-nor** [comment]
Have replaced the beam.conf file by the version from master branch.

**Comment by colinsheppard** [comment]
test!

File: src/main/scala/beam/utils/ScenarioReaderCsv.scala [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+object ScenarioReaderCsv {
+  private val logger = LoggerFactory.getLogger(this.getClass)
+
+  var vehicleCounter = 1;
+
+  def readPersonsFile(
+    filePath: String,
+    population: Population,
+    modes: String
+  ): TrieMap[Id[Household], ListBuffer[Id[Person]]] = {
```
Comment (REASY) [inline src/main/scala/beam/utils/ScenarioReaderCsv.scala]: Is there any particular reason to use `TrieMap` from `scala.collection.concurrent.TrieMap`? I don't see any concurrent read/write there.

File: src/main/scala/beam/utils/ScenarioReaderCsv.scala [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+        case None =>
+      }
+    }
+
+    logger.info("Reading Households...")
+    ScenarioReaderCsv.readHouseholdsFile(
+      householdFilePath,
+      scenario,
+      beamServices,
+      householdPersons.par,
```
Comment (REASY) [inline src/main/scala/beam/utils/ScenarioReaderCsv.scala]: Why make it as a parallel collection here? I don't see any `map` or other functions on these collections. I can see that it's used only for lookup (`.get`), so it won't give any benefit.

**Comment by REASY** [comment]
test!

**Comment by REASY** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Comment by colinsheppard** [comment]
test!

**Pull request merged**